macros/SpecName.ejs

```diff
@@ -39,6 +39,10 @@ var lang = env.locale;
 // * Please use HTTPS links instead of HTTP links when possible.
 
 var specList = {
+    'AcceleratedShape': {

```

Comment:
Is "Shape Detection API" a better name here (or more aligned with how the rest of the file names things)?